README.md

```diff
@@ -41,8 +41,8 @@ $ mix bootleg.init
 # config/deploy.exs
 use Bootleg.Config
 
-role :build, "your-build-server.local", user: "develop", password: "bu1ldm3", workspace: "/some/build/workspace"
-role :app, ["web1", "web2", "web3"], user: "admin", password: "d3pl0y", workspace: "/var/myapp"
+role :build, "your-build-server.local", user: "develop", identity: "my/sshkey.pri", workspace: "/some/build/workspace"
+role :app, ["web1", "web2", "web3"], user: "admin", identity: "my/sshkey.pri", workspace: "/var/myapp"
```

Comment:
How about `~/.ssh/id_deploy_rsa` as it's more realistic they'd have the key outside the project repo?

Reply:
hah, yeah, good point. updating...

Reply:
updated ⚡️